manifests/d/DoltHub/Dolt/0.26.11/DoltHub.Dolt.yaml

```diff
@@ -0,0 +1,24 @@
+PackageIdentifier: DoltHub.Dolt
+Publisher: DoltHub
```

Comment:
```suggestion
Publisher: Liquidata
```

Reply:
Company is renamed, now it is called  DoltHub, Inc.

Reply:
Dosn't matter when the installer shows the `Publisher` as `Liquidata`